tests/testthat/test-label-bytes.R

```diff
@@ -20,7 +20,7 @@ test_that("compatible with scale argument", {
 })
 
 test_that("errors if unknown unit", {
-  expect_error(label_bytes("unit")(0), "valid unit")
+  expect_error(label_bytes("unit")(0), "must be one of")
```

Comment:
let's make this a snapshot test while we touch it

Reply:
Yep, agreed